openassessment/data.py

```diff
@@ -1577,3 +1579,122 @@ def get_file_uploads(self, missing_blank=False):
                 files.append(file_upload)
             self.file_uploads = files
         return self.file_uploads
+
+
+def score_type_to_string(score_type):
+    """
+    Converts the given score type into its string representation.
+    """
+    SCORE_TYPE_MAP = {
+        PEER_TYPE: "Peer",
+        SELF_TYPE: "Self",
+        STAFF_TYPE: "Staff",
+        }
```

Comment:
why don't we use an enum or something similar

Reply:
Use an enum when you have a fixed and limited set of values that represent a specific data type, and you want to ensure their validity.
Use a mapping function when you need to perform more dynamic transformations or when the values are not fixed, and you want to provide a default value in case the input value is not found in the mapping.

The values for *_type come from here: openassessment/assessment/score_type_constants.py.

in order to pass this function to enun i thnk it would be something like this

```
from enum import Enum

class ScoreType(Enum):
    PEER_TYPE = "Peer"
    SELF_TYPE = "Self"
    STAFF_TYPE = "Staff"

def score_type_to_string(score_type):
    """
    Converts the given score type enum into its string representation.
    """
    return score_type.value if score_type in ScoreType else "Unknown"
```

I don't see any advantages in doing it.